project/src/components/main-page/main-page.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PlaceCard from '../place-card/place-card.jsx';
+
+function MainPage({offersAmount}) {
+  const getOffersAmount = () => new Array(offersAmount).fill().map((item, i, id=i) => <PlaceCard key={id} />);
```

Comment:
Запись не совсем хорошая. 
1. Первый шаг функции getOffers возвращает офферы, на текущий момент это набор липовых объектов только с одним полем id.
2. Второй шаг ты уже мапишь полученные офферы и выводишь разметку
```suggestion
  const getOffersAmount = () => new Array(offersAmount).fill().map((item, i) => ...);
  const offers = getOffersAmount(offersAmount);
  const renderCards = () => offers.map(({ id }) => <PlaceCard key={id} />)
```